install/install.sh

```
@@ -146,6 +169,65 @@ clines(){
+
+eth0_config(){
+	sudo nmcli con mod "Wired connection 1"   ipv4.method manual   ipv4.addresses 169.254.1.3/24   ipv4.gateway 169.254.1.1
+	sudo nmcli con mod "Wired connection 1" ipv4.route-metric 800
+	sudo nmcli connection down "Wired connection 1" && sudo nmcli connection up "Wired connection 1"
+}
+
+loveletter_extraction(){
+	#sudo hostnamectl set-hostname "$Boat"
+ 	clean_dir "$repo_extraction"
+	clone_repo $extract_tag $repo_extraction
+	cd LoveLetterExtraction
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)